packages/twenty-front/src/modules/command-menu/hooks/useCommandMenu.ts

```diff
@@ -36,6 +37,12 @@ export const useCommandMenu = () => {
           removeFocusItemFromFocusStackById({
             focusId: SIDE_PANEL_FOCUS_ID,
           });
+          set(
+            contextStoreCurrentObjectMetadataItemIdComponentState.atomFamily({
+              instanceId: CommandMenuPages.MergeRecords,
```

Comment:
I believe we already have what we need in contextStoreCurrentObjectMetadataItemIdComponentState with instanceId: COMMAND_MENU_COMPONENT_INSTANCE_ID

checking